packages/transport/src/car/codec.js

```diff
@@ -110,17 +110,16 @@ export const decode = async (bytes) => {
 
 /**
  * @param {Uint8Array} bytes
- * @param {{hasher?: API.MultihashHasher }} [options]
+ * @param {{hasher?: API.UCAN.MultihashHasher }} options
  */
 export const link = async (bytes, { hasher = sha256 } = {}) =>
-  /** @type {UCAN.Link<Model, typeof code, number> & import('multiformats').CID} */
+  /** @type {UCAN.Link<any, number>} */
   (createLink(code, await hasher.digest(bytes)))
 
 /**
  * @param {Partial<Model>} data
- * @param {{hasher?: API.MultihashHasher }} [options]
  */
-export const write = async (data, { hasher = sha256 } = {}) => {
+export const write = async (data) => {
```

Comment:
We should instead pass hasher into a link method

Reply:
did understand this one